appserver/batch/hazelcast-jbatch-store/pom.xml

```diff
@@ -53,19 +52,17 @@
         <dependency>
             <groupId>com.ibm.jbatch</groupId>
             <artifactId>com.ibm.jbatch.container</artifactId>
-            <version>${com.ibm.jbatch.container.version}</version>
             <type>jar</type>
         </dependency>
         <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast</artifactId>
-            <version>3.3.3</version>
+            <version>${hazelcast.version}</version>
```

Comment:
Does this actually need the version as it should come form the parent pom?

Reply:
Updated the dependency-management of parent module.